README.md

```diff
@@ -111,7 +111,14 @@ See https://github.com/simnalamburt/homebrew-x for the further information of th
 #### Install Lima
 
 - Download the binary archive of Lima from https://github.com/lima-vm/lima/releases ,
-and extract it under `/usr/local` (or somewhere else).
+and extract it under `/usr/local` (or somewhere else). For instance:
+
+```bash
+brew install jq
+sudo -i
+VERSION=$(wget -cq https://api.github.com/repos/lima-vm/lima/releases/latest -O - | jq -r .tag_name )
+wget -cq https://github.com/lima-vm/lima/releases/download/$VERSION/lima-${VERSION:1}-`uname`-`uname -m`.tar.gz -O - | tar -xzC /usr/local
```

Comment:
Why `-c`?

Reply:
Sorry, removed `-c`.
Also, the current `tar -xz` line will have error/warning:
```
./: Can't restore time
tar: Error exit delayed from previous errors.
```
It should be related to bsdtar, but I could not get rid of it w/o redirect stderr, do you have idea on this, please?

Reply:
Added a follow-up commit `0969544aa1d45f617923f09cf7abdc33cef14474` for the time issue